.github/workflows/project.yml

```diff
@@ -169,3 +169,36 @@ jobs:
       - uses: ./.github/actions/node/setup
       - uses: ./.github/actions/install
       - run: node scripts/verify-ci-config.js
+
+  actionlint:
```

Comment:
Wouldn't it be prudent to put this in its own yaml file, so that the risk of a syntax error in that yaml file was as low as possible? That way a syntax error in `project.yml` would be detectable

Reply:
Sure I can do that.

Reply:
Done